src/components/AdvancedOptions.tsx

```diff
@@ -230,7 +230,7 @@ export default function AdvancedOptions(): ReactElement {
               <input
                 type="text"
                 className={classNames({
-                  [styles.selected]: gasPriceSelected === GasPrices.Custom,
```

Comment:
What is the advantage of moving away from the `styles` class?

Reply:
makes testing a little easier, but I don't think it's worth it